qml/Keyboard.qml

```diff
@@ -272,7 +274,7 @@ function calculateSize()
     var newHeight;
     if( Settings.tabletUi ) {
         if( isLandscape ) {
-            newHeight = fullScreenItem.height * UI.tabletKeyboardHeightLandscapeM
+            newHeight = fullScreenItem.height * UI.tabletKeyboardHeightLandscapeL
```

Comment:
We should find a way to adapt that w.r.t. the current selected layout option on Dismiss Key popup, but I don't have a proposal yet.